packaging/linux/flatpak/scripts/install-mouse.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+UDEV=$(cat /app/share/sunshine/udev/rules.d/85-sunshine.rules)
+flatpak-spawn --host pkexec sh -c "usermod -a -G input $USER && echo '$UDEV' > /etc/udev/rules.d/85-sunshine.rules"
```

Comment:
Was this tested? Does `$USER` evaluate properly?

Reply:
Yes